packages/tabs/src/tab-nav.vue

```diff
@@ -113,11 +113,6 @@
         }
       }
     },
-
-    updated() {
-      this.update();
```

Comment:
This will lead to a bug that the nav__active-bar do not scroll to the right place